### DISCUSSION TITLE
Changing the IP-Adress

### DISCUSSION THREAD

**Discussion by marslbl** [Seji64/LAPS-WebUI#22] [General]
Hello, 

I have now set up the LAPS WebUI and it is also available at http://127.0.0.1:5000. However, I would now like to change the address and port so that I can access the LAPS WebUI from the internal network via the server name and another port. For example: http://servername:11115. Unfortunately I have not found anything about this, is it possible to adapt it using the existing software or are further programs/settings required?

Thanks in advance!

**Comment by Seji64** [comment]
See #5
